### ISSUE TITLE
Agree on overall design

### ISSUE THREAD

**Issue by VirtualDOMinic** [fac-14/tech-access#11]
At least initial agreement.

Should produce a markdown summary/guide to go from

25-60 minutes

**Comment by MissArray** [comment]
I tested these with Color Cube; all three have good accessibility rates (for blue-black text against a coloured background).

https://oomphinc.github.io/colorcube/#results-content

RATIOS - ochre / cadet blue / blue-black: #4986A1
#4986A1
#FDD05C
#0D1B30
#4986A1
#C9DEE7
#AFD2E2

RATIOS - PALETTE 2

#9CACCC
#F3D670
#CCD5E7
#B9C3D7
#0D1B30

PALETTE 3 - apple green / lavender-grey / blue-black text

#CAEFAC
#DDCADF
#EBE2EC
#0D1B30
